sql-plugin/src/main/scala/com/nvidia/spark/rapids/aggregate.scala

```diff
@@ -662,15 +577,15 @@ case class GpuHashAggregateExec(
     // - Partial, PartialMerge mode: we use the inputProjections or distinct update expressions
     //   for Partial and non distinct merge expressions for PartialMerge.
     // - Final mode: we pick the columns in the order as handed to us.
-    val boundInputReferences = if (uniqueModes.contains(PartialMerge)) {
+    val boundInputReferences = if (modeInfo.uniqueModes.contains(PartialMerge)) {
```

Comment:
minor nit, we have  `hasFinalMode` and `hasCompleteMode` and others, can we add a `hasPartialMerge`?